File: SDKs/googleapis.com/forms/v1/python/src/sdk/models/shared/response.py
import dataclasses
from datetime import date, datetime
from marshmallow import fields
import dateutil.parser
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import createitemresponse as shared_createitemresponse


@dataclass_json
@dataclasses.dataclass
class Response:
    r"""Response
    A single response from an update.
    """
    
    create_item: Optional[shared_createitemresponse.CreateItemResponse] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('createItem') }})
    

File: SDKs/googleapis.com/forms/v1/java/lib/src/main/java/openapisdk/models/shared/Response.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * Response
 * A single response from an update.
**/
public class Response {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("createItem")
    public CreateItemResponse createItem;
    public Response withCreateItem(CreateItemResponse createItem) {
        this.createItem = createItem;
        return this;
    }
}